restapi/user_objects.go

```diff
@@ -68,11 +69,16 @@ func registerObjectsHandlers(api *operations.ConsoleAPI) {
 			return user_api.NewDownloadObjectDefault(int(err.Code)).WithPayload(err)
 		}
 		return middleware.ResponderFunc(func(rw http.ResponseWriter, _ runtime.Producer) {
-			x, err := io.Copy(rw, resp)
-
-			fmt.Println(x)
-			fmt.Println(err)
-
+			stat, err := resp.(*minio.Object).Stat()
```

Comment:
```suggestion
			stat, err := resp.(*minio.Object).Stat()
```

Do not ignore the error, log it or return it to the handler

Reply:
solved